docs/manage/manage-subscription/create-manage-orgs-service-providers.md

```diff
@@ -63,6 +63,12 @@ The credits associated with the trial org allocations don’t impact the parent
 
 You can upgrade a trial org by editing the org and changing the Plan Type. When you upgrade a POV Trial org, if the org is in a different Sumo Logic deployment from the parent org, the Credits çalculator may add a deployment factor, which is a percentage upcharge that varies by deployment.
 
+The plan gets downgraded to **Free** once the 45-day POV trial period is over. The data will be deleted post seven days after your POV trial expiration date, which can create issues when you request a re-activation. So, to overcome this issue, Sumo Logic now alerts you in three different places as shown below on the platform about the expiration date.
```

Comment:
```suggestion
The plan gets downgraded to **Free** once the 45-day POV trial period is over. The data will be deleted seven days after your POV trial expiration date, which can create issues when you request a re-activation. To prevent this from happening, Sumo Logic now alerts you in three different places as shown below on the platform about the expiration date.
```